lib/core/src/server/manager/manager-config.js

```diff
@@ -91,6 +92,17 @@ async function getManagerWebpackConfig(options, presets) {
   }
   if (autoRefs || definedRefs) {
     entries.push(path.resolve(path.join(options.configDir, `generated-refs.js`)));
+
+    // verify the refs are publicly reachable, if they are not we'll require stories.json at runtime, otherwise the ref won't work
+    await Promise.all(
+      Object.entries(refs).map(async ([k, value]) => {
+        const { ok } = await fetch(`${value.url}/iframe.html`).catch((e) => ({
+          ok: false,
+        }));
```

Comment:
I'm not sure about mixing promises and await. How about:

```suggestion
        let type = 'unknown';
        try {
          const { ok } = await fetch(`${value.url}/iframe.html`);
          if (ok) {
             type = 'server-checked';
          }
        } catch (err) {
          // pass
        }
```

Reply:
I'd rather have the above personally. I don't think mixing async/await with promises is bad.

But maybe if I extract the fetch and catch into a function you wouldn't mind the mixing of async/await and promises?